File: backend/components/metiers/threed/tests_centralises/fixtures/helpers/rgpd/test_helper_rgpd.js
// test_helper_rgpd.js – Tests helpers rgpd (JS)
const { anonymizeData } = require('../../../../fixtures/helpers/rgpd/helper_rgpd');
test('anonymizeData', () => {
  expect(anonymizeData({ a: 1, b: 2 })).toEqual({ a: '***', b: '***' });
});

File: backend/components/metiers/threed/tests_centralises/fixtures/helpers/rgpd/test_helper_rgpd.py
# test___init__.py – Tests helpers rgpd (Python)

import importlib.util
import os

spec = importlib.util.spec_from_file_location(
    "helper_rgpd",
    "/workspaces/dihya.io/backend/components/metiers/threed/fixtures/helpers/rgpd/helper_rgpd.py",
)
helper_rgpd = importlib.util.module_from_spec(spec)
spec.loader.exec_module(helper_rgpd)


def test_anonymize_data():
    data = {"a": 1, "b": 2}
    anonymized = helper_rgpd.anonymize_data(data)
    assert anonymized == {"a": "***", "b": "***"}
